src/lib/SeamComponents.tsx

```diff
@@ -1,5 +1,11 @@
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query'
```

Comment:
Was there a reason for this rename? I think we should keep the name `SeamProvider`. The file rename to match the component name is ok though.

Reply:
Not really, just wanted to get some thoughts. Updated in https://github.com/seamapi/react/pull/47/commits/cf2023f528bba35db575bd5a17b3a48dd56c0fc1

Reply:
Ok, I think SeamProvider is more ubiquitous: people instantly recognize the provider pattern that way.